pkg/deploy/service.go

```diff
@@ -266,7 +283,33 @@ func getFileNames(manifestPath model.ManifestPath) (error, []string) {
 	return err, fileNames
 }
 
-func CreateDeploymentManifests(manifests *[]string) []de.Manifest {
+func containsDeployment(manifest string) (bool, error) {
```

Comment:
stolen from Deploy Engine. But this will now work with multiple k8 objects in a single manifest